tests/test_events.py

```diff
@@ -29,42 +28,45 @@ async def _ignore_test_consoleLog_logEntryAddedEventEmitted(websocket):
             "expression": "console.log('some log message')",
             "target": {"context": context_id}}})
 
+    # The order of the two responses are not guarenteed
+    resp1 = await read_JSON_message(websocket)
+    resp2 = await read_JSON_message(websocket)
+    if "id" in resp1:
+        resp1, resp2 = resp2, resp1
+
     # Assert "log.entryAdded" event emitted.
-    resp = await read_JSON_message(websocket)
     recursiveCompare(
-        resp,
+        resp1,
```

Comment:
`recursiveCompare` accepts `actual` and then `expected` arguments, not vice versa (but sometimes I misused it).

Reply:
updated the order of the arguments in the test.